tutor/src/components/annotations/annotation.jsx

```diff
@@ -227,10 +227,10 @@ export default class AnnotationWidget extends React.Component {
     for (const other of this.annotationsForThisPage) {
       if (selection.referenceElementId === other.referenceElementId) {
         if (selection.start >= 0 &&
-          (other.selection.start > selection.start &&
-            other.selection.start < selection.end) ||
-          (other.selection.end > selection.start &&
-            other.selection.end < selection.end)) {
+          (other.selection.start >= selection.start &&
+            other.selection.start <= selection.end) ||
+          (other.selection.end >= selection.start &&
+            other.selection.end <= selection.end)) {
 
```

Comment:
hah, that completely makes sense and explains why the first/lat character would be unhighlightable at times.,